optimus/io/properties.py

```diff
@@ -0,0 +1,59 @@
+from enum import Enum, unique
+
+
+@unique
+class DriverProperties(Enum):
+    """Enum holding all driver properties."""
+
+    CASSANDRA = {
+        "name": "cassandra",
+        "port": 9042,
+        "java_class": "org.apache.spark.sql.cassandra",
+        "table_name": ""
+    }
+    MYSQL = {
+        "name": "mysql",
+        "port": 3306,
+        "java_class": "",
+        "table_name": "table_name"
+    }
+    ORACLE = {
+        "name": "oracle",
+        "port": 1521,
+        "java_class": "oracle.jdbc.OracleDriver",
+        "table_name": "table_name"
+    }
+    POSTGRESQL = {
+        "name": "postgresql",
+        "port": 5432,
+        "java_class": "org.postgresql.Driver",
+        "table_name": "table_name"
+    }
+    SQLSERVER = {
+        "name": "sqlserver",
+        "port": 1433,
+        "java_class": "com.microsoft.sqlserver.jdbc.SQLServerDriver",
+        "table_name": "TABLE_NAME"
+    }
+    SQLITE = {
+        "name": "sqlite",
+        "port": 0,
+        "java_class": "org.sqlite.JDBC",
+        "table_name": "name"
+    }
+    PRESTO = {
+        "name": "presto",
+        "port": 8080,
+        "java_class": "com.facebook.presto.jdbc.PrestoDriver",
+        "table_name": ""
+    }
+    REDSHIFT = {
+        "name": "redshift",
+        "port": 5439,
+        "java_class": "",
```

Comment:
Should we use ```com.amazon.redshift.jdbc.Driver``` for the java class?

Reply:
Currently we don't set that option for Redshift (line 154 jdbc.py):

```
# Driver
if self.db_driver == DriverProperties.ORACLE.value["name"] \
        or self.db_driver == DriverProperties.POSTGRESQL.value["name"] \
        or self.db_driver == DriverProperties.PRESTO.value["name"]:
    conf.option("driver", self.driver_option)```

We could definitely add it though, let me know your thoughts (unfortunately now sure how to really test Redshift)